<launch>
    <!-- <node name="xtark_driver" pkg="xtark_driver" type="xtark_driver" output="screen" ns="follower1"  respawn="true"></node> -->
    <node pkg="tf_follower"  type="tf_follower_broadcaster"   name="pub1"   args="follower1"    output="screen"></node>
    <node pkg="tf_follower"  type="tf_listener"  name="listener1"  ></node>
</launch>